pkg/cluster/cluster.go

```diff
@@ -209,7 +214,7 @@ func CreateCluster(cmv1Client *cmv1.Client, config Spec, dryRun bool) (*cmv1.Clu
 
 	if config.CCS.Enabled {
 		clusterBuilder = clusterBuilder.CCS(cmv1.NewCCS().Enabled(true))
-		if config.Provider == "AWS" {
+		if config.Provider == ProviderAWS {
```

Comment:
not a blocker, but I would put this in a `switch`, failing if none of the providers are valid.

Reply:
Good idea, done